### DISCUSSION TITLE
Pinocchio + {PyTorch,Tensorflow}

### DISCUSSION THREAD

**Discussion by jcarpent** [stack-of-tasks/pinocchio#1508] [Ideas]
Make the full Pinocchio computational graph compliant with {PyTorch,Tensorflow}.

**Comment by zongmianli** [comment]
Is there any update on this?  I'm using [pytorch-kinematics](https://github.com/UM-ARM-Lab/pytorch_kinematics) for batch processing on GPU. But I'm considering switching to pinocchio for the dynamics. Thanks!
